schema/gen/go/testStructsContainingMaybe_test.go

```diff
@@ -335,14 +40,119 @@ func TestStructsContainingMaybe(t *testing.T) {
 		}),
 	))
 
+	// There's a lot of cases to cover so a shorthand labels each case:
+	//  - 'v' -- value in that entry
+	//  - 'n' -- null in that entry
+	//  - 'z' -- absent entry
+	specs := []testcase{
+		{
+			name:     "all fields set -- vvvvv",
```

Comment:
test names, and by extension sub-test names, should generally be camelcase. So you should do `SomeTest/SomeSubtest` instead of `some test/some subtest`, particularly because spaces don't play nice with the input/output and get replaced by other characters.

I think symbols are OK if needed, so you could do something like `AllFieldsSet-vvvvv`.

Reply:
Is this an assumption built into tooling, or just a convention that's good to adhere to just because it's a convention? (Not being disagreeable, simply interested).

Reply:
Yeah, it's a convention - though driven by the tooling as well, because the test names can't contain whitespace and thus it gets replaced in the output of `go test`. It's also encouraged by the upstream packages; you can search for "naming convention" under https://golang.org/pkg/testing/#hdr-Examples.

Reply:
IIUC, any funky characters get replaced by underscores.  I typically don't really mind this; the result still reads clearly enough in the terminal.

In this case the resulting "`_--_`" is particularly nasty, though, so yeah, I'll rejigger these.

Reply:
it's just easier for everyone if the names are consistent in the output and source code, and it's also more consistent with how most people tend to write Go tests :)